main.go

```diff
@@ -120,6 +149,41 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "Gatekeeper")
 		os.Exit(1)
 	}
+
+	var wg sync.WaitGroup
+
+	dynamicClient := dynamic.NewForConfigOrDie(mgr.GetConfig())
+	wg.Add(1)
+
+	c := ConstraintStatusInstaller{isInstalled: false}
+
+	go c.watchGatekeeperInstall(mainCtx, dynamicClient, cfg, ctrl.Options{
+		Scheme: scheme,
+		Metrics: server.Options{
+			BindAddress: ":8083",
+		},
+		HealthProbeBindAddress: ":8084",
+		LeaderElection:         enableLeaderElection,
+		LeaderElectionID:       "5ff985ccc.constraintstatuspod.gatekeeper.sh",
+		Cache: cacheRuntime.Options{
+			ByObject: map[client.Object]cacheRuntime.ByObject{
+				&v1beta1.ConstraintPodStatus{}: {
+					Transform: func(obj interface{}) (interface{}, error) {
+						constraintStatus := obj.(*v1beta1.ConstraintPodStatus)
+						// Only cache fields that are utilized by the controllers.
+						guttedObj := &v1beta1.ConstraintPodStatus{
+							TypeMeta:   constraintStatus.TypeMeta,
+							ObjectMeta: constraintStatus.ObjectMeta,
```

Comment:
It'd be nice if this only included the name, namespace, and labels since that's all you use. The annotations can be large (e.g. `last-applied`) and the managed fields can take up quite a bit of space as well.